tests/routes/test_peers.py

```
@@ -527,6 +527,12 @@ def test_get_peers_with_complex_filter(
     )
     assert response.status_code == 200
     data = response.json()
+    # for each peer returned, assert that the metadata["index"] fits our filter
+    for peer in data["items"]:
+        assert (
+            peer["metadata"].get("index", 0) >= 1
+            and peer["metadata"].get("type", "") == "test"
+        )
```

Ruff (0.12.2)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)